### DISCUSSION TITLE
fluid crafting pipe(s)/module(s)

### DISCUSSION THREAD

**Discussion by loordgek** [RS485/LogisticsPipes#1596] [Ideas]
fluid crafting pipe(s)/module(s)

item into fluid crafter  
for example
thermal expansion magma crucible put  x amount of redstonedust in and get x amount destabilized redstone

fluid into fluid crafter 
for example
tinkers construct smeltery  x amount of molten Aluminum in, and x amount of molten Copper and get x amount of molten Liquid Aluminum Brass

**Comment by SonOfTheStars** [comment]
Sattelite pipes do your Trick ;)

**Comment by loordgek** [comment]
yea but it is a pain in the ass to set that up

**Comment by bziemons** [comment]
@Bitterholz nope, there are no fluid results. So crafting with fluids work, but crafting fluids does not.
There are workarounds though, because you always can craft an item in those two cases though.

**Comment by SonOfTheStars** [comment]
@theZorro266 I did it before, I had a Buffertank and Produced new fluid on Usage wqich got routed into the Tank ;)
